### ISSUE TITLE
`meteor mongo stop; meteor mongo start` killed my production database?

### ISSUE THREAD

**Issue by DanielDornhardt** [zodern/meteor-up#1022]
(TL;DR at the bottom)

Hi everybody,

thank you thank you thank you everybody for this great tool.

Unfortunately, a few weeks ago I had a live instance of a project running extremely hot on CPU and when checking `top` and some mongo running queries I had a sneaking suspicion that it might actually be the mongodb instance which ran hot.

(Later it turned out it probably wasn't, the load stemmed mostly from my js / publication code).

Anyway, because I had restarted the app a few times already using

    mup restart

(There was some action going on in the app at the time, there was a live event and people were using the app), I thought it might be quicker to just try and restart just the mongodb instance. As I said, I wasn't sure whether it was node or mongo which was running hot.

So, just because of habit i ran `mup help` and saw that there was a `mup mongo` command. A quick `mup mongo help` showed there to be two commands useful for my purpose: `stop` and `start`.

So I ran first `mup mongo stop`, waited for mup to finish doing it's thing, and then `mup mongo start`.

At first I thought: Neat! :D Good job Daniel! Carry on! :D

... but then, half a minute later, my phone rang.

It was the client... and all of his live event data seemed to be gone... I don't know what happened exactly. And that is why I am writing this.

What I could see is that my insert-some-fixture-data-into-the-database-if-there-is-nothing-in-there - script had run and inserted the default data.

But that means that everything else before has had to be empty. Or maybe Meteor just thought so for a moment and created new collections for everything? I am not sure how that could have happened.

Something must have gone terribly wrong while using `mup mongo stop` and `mup mongo start`.

Is it possible that they shouldn't be used while meteor is running? Is there a chance that the scripts are only meant to be ran whilst meteor is down?

Maybe there could be additional information and/or warnings in the help as well as in other places what `mup mongo` in general and `start/stop` specifically will do. I think I searched the docs and didn't find any infos for those commands... but maybe I just didn't see it.  

Anyway, I thought I really should write all of this down, sorry, it got a bit long, but I am trying to tell the story as good as I could so the whole process is clear.

TL;DR:

I ran `mup mongo stop` - waited - and then `mup mongo start` on a running instance. After that my collections were empty.

Not good!

Have a nice one everybody! :)

**Comment by promentol** [comment]
@DanielDornhardt did you find a solution at that time?

**Comment by DanielDornhardt** [comment]
Hi @promentol , unfortunately I didn't, except to never run mup mongo stop / start again...
